File: Examen1/2026_1/Resolucion/ExamenParcialCSharp/GestionAlumnosBusiness/Alumnos/BOI/IAlumnoBO.cs
using GestionAlumnosBusiness.BO;
using GestionAlumnosModel.Alumno;

namespace GestionAlumnosBusiness.Alumnos.BOI
{
    public interface IAlumnoBO : IBaseBO<Alumno>
    {
        int BuscarPorCodigo(string codigo);
        List<Alumno> BuscarPorNombreApellido(string texto);
    }
}

File: lab11Calificado/Lab11_PLANTILLA/TestSoftSolution/TestSoftBusiness/src/main/java/pe/edu/pucp/testsoft/business/alumno/bo/IAlumnoBO.java
package pe.edu.pucp.testsoft.business.alumno.bo;

import pe.edu.pucp.testsoft.business.bo.IBaseBO;
import pe.edu.pucp.testsoft.model.Alumno;
import java.util.List;

public interface IAlumnoBO extends IBaseBO<Alumno> {
    List<Alumno> buscarPorNombreApellido(String texto) throws Exception;
}
